packages/oscal-react-library/src/components/OSCALControl.tsx

```diff
@@ -74,17 +102,25 @@ function ControlsList(props) {
           modificationSetParameters={modificationSetParameters}
           onRestError={onRestError}
           onRestSuccess={onRestSuccess}
-          parameters={listControl.params}
           partialRestData={partialRestData}
           urlFragment={urlFragment}
           fragmentPrefix={fragmentPrefix}
+          withdrawn={props.withdrawn}
         />
       ))}
     </div>
   );
+};
+
+export interface OSCALControlProps extends ControlsListProps, AnchorLinkProps {
+  listItemOpened?: boolean;
+  isItemNavigatedTo?: boolean;
+  showInList?: boolean;
+  previousHandledFragment?: string;
+  setPreviousHandledFragment?: (fragment: string) => void;
```

Comment:
I think this stuff about being open and about fragments probably should be an interface if it gets used in any other components ever...

Reply:
This should probably be done in a separate PR.